deployment/ansible/roles/cac-tripplanner.otp-data/templates/otp.conf.j2

```diff
@@ -0,0 +1,13 @@
+## NOTICE: This file is written by ansible, and any changes made here will be overwritten on next provision.
+#           Modify azavea.opentripplanner/templates/otp.conf.j2 to make changes stick.
```

Comment:
`cac-trippplanner.otp-data`?

Although, on a related note, I'd be ok with modifying the azavea opentripplanner role to require you to override a template, since that's way more flexible.